bootstrap-salt.sh

```diff
@@ -1874,8 +1894,13 @@ __apt_get_upgrade_noinput() {
 __apt_key_fetch() {
     url=$1
 
-    # shellcheck disable=SC2086
-    __wait_for_apt apt-key adv ${_GPG_ARGS} --fetch-keys "$url"; return $?
+    tempfile=__temp_gpg_pub || return 1
```

Comment:
The shell is very weird "programming language" :smile: 
You can't just assign an output of the function to a variable like that. The Command Substitution must be used: `$(...)`

Reply:
@vutny you've got so much patience and me, not enough coffee. thanks!

Reply:
No worries, @greut